doc/sphinx/practical-tools/coqide.rst

```diff
@@ -263,8 +261,8 @@ for the ∀ symbol. A list of symbol codes is available at
 An alternative method which does not require to know the hexadecimal
 code of the character is to use an Input Method Editor. On POSIX
 systems (Linux distributions, BSD variants and MacOS X), you can
-use `uim` version 1.6 or later which provides a :math:`\LaTeX`-style input
-method.
+use `uim` version 1.6 or later which provides a
+:math:`\text{\LaTeX}`-style input method.
```

Comment:
Can't we just say LaTeX? This sounds overly complicated just to get a fancy font.